### PR TITLE
Added notification-badge to automation-tab

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | https://github.com/massiveart/husky/pull/755
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR introduces notification-badges for automation-badges.

### CODE REVIEW

File: src/Sulu/Bundle/AutomationBundle/Admin/AutomationContentNavigationProvider.php [hunk 1]
```diff
@@ -62,9 +74,17 @@ public function getNavigationItems(array $options = [])
         $automation->setPosition($this->position);
         $automation->setAction('automation');
         $automation->setComponent('automation-tab@suluautomation');
-        $automation->setComponentOptions(['entityClass' => $this->entityClass]);
         $automation->setDisplay(['edit']);
 
+        $componentOptions = ['entityClass' => $this->entityClass];
+        if (array_key_exists('id', $options)) {
+            $locale = array_key_exists('locale', $options) ? $options['locale'] : null;
+            $futureTasks = $this->taskRepository->countFutureTasks($this->entityClass, $options['id'], $locale);
```
Comment (danrot) [inline src/Sulu/Bundle/AutomationBundle/Admin/AutomationContentNavigationProvider.php]: The variable name sounds like an array of tasks. Would rename to something like `futureTaskCount`.

File: src/Sulu/Bundle/AutomationBundle/Resources/public/js/components/automation-tab/main.js [hunk 2]
```diff
@@ -216,14 +218,17 @@ define([
         deleteTasks: function(ids) {
             return manager.deleteItems(ids).then(function() {
                 _.each(ids, function(id) {
+                    this.notificationBadge--;
```
Comment (danrot) [inline src/Sulu/Bundle/AutomationBundle/Resources/public/js/components/automation-tab/main.js]: `--this.notificationBadge`

File: src/Sulu/Bundle/AdminBundle/Navigation/ContentNavigationItem.php [hunk 3]
```diff
@@ -90,6 +90,11 @@ class ContentNavigationItem
      */
     private $displayConditions;
 
+    /**
+     * @var int
+     */
+    private $notificationBadge;
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Navigation/ContentNavigationItem.php]: For what is this needed? Doesn't seem to be used anywhere.

Comment (wachterjohannes) [inline src/Sulu/Bundle/AdminBundle/Navigation/ContentNavigationItem.php]: it will be needed for initial state of notification

**Comment by wachterjohannes** [comment]
@danrot fixed

**Comment by danrot** [comment]
There is an issue with adding tasks in the past, they will appear in the wrong list, and the badge does update incorrectly.

But since this is an extreme edge case (it doesn't make sense to do it this way) I will merge anyway after tests pass. Might be corrected in a separate PR.
